### PR TITLE
Portfolio project styling

### PR DESCRIPTION
**Pull request by Samuelwanza**
I have styled the header section of the mobile version of the portfolio:
-I have styled the appbar/header
-Added the social media icons
-Changed the logo in the readme file

### CODE REVIEW

**Comment by NahnahAJ** [comment]
## Project Status: INVALIDATED :red_circle: 
Hi @Samuelwanza , 

While you made a great effort in this project, I can't go ahead and review your code.


## Invalid Code Review Request
- [ ]  This is because your README contains the right details. However, you have altered the original `Table of contents` of the original [Microverse README Template](https://github.com/microverseinc/readme-template)](https://github.com/microverseinc/readme-template).
- [ ]  I also noticed that you removed some of the headers from the [Microverse README Template](https://github.com/microverseinc/readme-template)](https://github.com/microverseinc/readme-template). This makes your project incomplete as every section of the readme template is required except for the `FAQ section` which is optional. So please make sure to add all the missing` headers/sections ` in the README template to your project and fill in all the necessary contents :+1:

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,178 @@
...
+
+### Deployment
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
```
Comment (skyv26) [inline README.md:114]: - [ ] Please remove the placeholder or comment from your `README.md` file. These comments or placeholders are just meant for guidance purposes while creating `documentation` of your project not for users to see.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,178 @@
...
+- **[Projects]**
+- **[Testimonials]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<p>This project is  not yet deployed</p>
```
Comment (skyv26) [inline README.md:69]: - [ ] Kindly remove this section if your project does not have a deployed version or put a notification telling users that the deployed version is not available yet.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,178 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+<p> To run the project, clone the repository to the choice directory and then launch vs code , install live server and launch it</p>
+
+### Prerequisites
```
Comment (skyv26) [inline README.md:79]: - Kindly add the proper `instructions` to help the user to understand how to use your project. Please consider the below snippet.

Screenshot from 2023-04-25 19-31-28

File: README.md [hunk 4]
```diff
@@ -1,2 +1,178 @@
...
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/Samuelwanza/portfolio_setup_and_mobile_first
```
Comment (skyv26) [inline README.md:97]: - [x] Please be more specific in your documentation which helps the others about how to use your project. Please consider the below example.

```suggestion
Clone this repository to your desired folder:

### Example commands:

<code>
  git clone git@github.com:HtetWaiYan7191/hello-microverse-project.git
  cd hello-microverse-project
</code>

File: README.md [hunk 5]
```diff
@@ -1,2 +1,178 @@
...
+```
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```
Comment (skyv26) [inline README.md:106]: - [ ] Kindly configure these sections to assist anyone using the repo, make it descriptive as possible so anyone without programming knowledge can run your application locally on their device

File: README.md [hunk 6]
```diff
@@ -1,2 +1,178 @@
...
+
+- **[Yes you can]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (skyv26) [inline README.md:174]: - [ ] Kindly add the correct path of the MIT `LICENSE` in your project, as I can see you have not added the path correctly.  Please look at the below suggestion to solve your problem.

``` suggestion
This project is [MIT](./MIT.md) licensed.
```

**Review by skyv26 — CHANGES_REQUESTED** [review]
## Hi @Samuelwanza , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Linters are green ✅ 
- Well-structured files and folder ✅

## Required Changes ♻️
- [ ] Kindly make your PR description more informative which helps any person to know what you did implement as a feature in the PR branch.

### Your PR 👤 
Screenshot from 2023-04-28 22-40-11

### Descriptive PR
Screenshot from 2023-04-28 21-07-13
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @skyv26 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.css [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+.header-herosection > h3 {
+  color: #7f8cff;
+  padding-left: 20px;
+}
+
+.social-media {
+  display: flex;
+  padding-left: 20px;
+}
```
Comment (franclobo) [inline index.css:75]: - **[OPTIONAL]**  Add the space at the end and the border radion on the left to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| image | image |

**Review by franclobo — APPROVED** [review]
Hi @Samuelwanza, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Correct Giyhub flow ✔️ 
- Descriptive PR ✔️ 
- Professional README file ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**